src/games/strategy/triplea/ui/MapData.java

```diff
@@ -204,11 +204,23 @@ private void loadTerritoryNames() {
     }
     for (final String name : m_centers.keySet()) {
       try {
-        final Image img = loadImage("territoryNames/" + name + ".png");
+        String normalizedName = name.replace(' ', '_');
+        String path = "territoryNames/" + normalizedName + ".png";
+        Image img = loadImage(path);
+
+        if(img == null ) {
+          path = "territoryNames/" + normalizedName + ".png";
```

Comment:
Should one of these be trying 'name' instead of 'normalizedName'? It appears its doing the same thing twice.